src/config.json

```diff
@@ -88,6 +91,7 @@
         "awe": "http://140.221.67.185:7080",
         "catalog": "https://ci.kbase.us/services/catalog",
         "cdmi": "https://kbase.us/services/cdmi_api/",
+        "cdn": "https://ci.kbase.us/cdn",
```

Comment:
That's pretty much a holdover from the pre-CI/Next/appdev days. We can probably lay it to rest now.